cmd/juju/backups/restore.go

```diff
@@ -88,10 +110,35 @@ func (c *restoreCommand) Init(args []string) error {
 			return errors.Trace(err)
 		}
 	}
+
 	return nil
 }
 
-func (c *restoreCommand) NewClient() (*backups.Client, error) {
+func (c *restoreCommand) modelStatus() (string, []base.ModelStatus, error) {
+	modelUUIDs, err := c.ModelUUIDs([]string{bootstrap.ControllerModelName})
+	if err != nil || len(modelUUIDs) != 1 {
```

Comment:
I think if ```len(modelUUIDs) != 1``` then ```Annotatef``` will return a nil error, we will get ```"", nil, nil```.

Reply:
yes, updated.